### ISSUE TITLE
wierd voltages 

### ISSUE THREAD

**Issue by KeLaiFu** [Andreas-Dorfner/Powerline-USB-C#1]
I've built one of these and I am getting -12 and 5v on the header, but 4.9v on the +12. I checked the power coming from pin 1 on the IC, and its 1.3v (?) but 4.6v on the VBUS pin on the other side of R1.

I'm using meanwell DC blocks and ICs from LCSC and with such a small BOM, I am at a loss...

Any ideas?

**Comment by Andreas-Dorfner** [comment]
I'm sorry your module isn't working as it should. The good thing is the voltages you measured mean that most of the module is ready to go. As for the rest:
The first thing to check is if the USB-C brick you are using supports 20V and the PD protocol(Other protocols are not supported). 
If that is the case, that would indicate a problem with the USB-C trigger Circuit around the CH224K.
I still need to build the USB-C version of this module jet, so I can't confirm if the problem is in the schematic or your particular build. However, as you are using genuine parts, I highly suspect that the default implementation in the CH224k datasheet is faulty. As a quick fix, if you supply the module with 20V through the USB-C port with a rigged cable (like a USB-C trigger cable) or a barrel Jack, it will work for now.
I will review the USB-C trigger section again and update you if I find an error.

**Comment by KeLaiFu** [comment]
OK, thanks for the update. the USB-C power source I am using states 5A / 20V and powers my new laptop without issues so I think that is ok. I tried a second one which is lower spec and the results were the same.

If you need me to test / try anything let me know.

**Comment by CIRCUITF0RGE** [comment]
Hey,

i got kinda also into power trouble...get way off readings only 5V seems around 4.96, so cool...but i used bigger 7812-R2000 bricks and maybe there is the prob...happy for any update or suggests...

**Comment by Andreas-Dorfner** [comment]
Sadly Uni has prevented me from tinkering with anything, so I am still not sure if that is the solution. But I would suggest injecting 20V into the USB-C port directly and not rely on the PD chip to work correctly. The rest of the circuit should work. I will update the repo as soon as I find a working solution.
Sorry for all the inconvenience.

**Comment by KeLaiFu** [comment]
has this been forgotten? I still have a bag of DC blocks and caps waiting for this one...

**Comment by Andreas-Dorfner** [comment]
Not at all, still lurking on my to do list. I implemented the fixed PD circuit in the schematic and did the PCB layout at the end of January, but I lack the time and resources at the time to test it.

**Comment by Andreas-Dorfner** [comment]
The module is fixed and working for me. The problem was the datasheet of the CH224K; the example circuits don't work because you need 3.3V. I updated the repo. @KeLaiFu  @stilltryin

**Comment by CIRCUITF0RGE** [comment]
thx
